Core/Repositories/Adapters/RequestFileSystemRepository.ts

```diff
@@ -4,13 +4,82 @@ import { Request } from '../../Models/request';
 
 class RequestRepository implements Port.IRequestRepository {
   private _requestsConfigFilePath: string =
```

Comment:
Doesn't related to this review but I would rename ```private _requestsConfigFilePath to private readonly REQUESTS_FILE_PATH``` - The capital letters indicates a const.

Reply:
Agree, done